crates/prover/src/tls/config.rs

```diff
@@ -100,11 +109,19 @@ impl ProverConfig {
     }
 
     pub(crate) fn ot_sender_setup_count(&self) -> usize {
-        ot_send_estimate(Role::Prover, self.max_sent_data, self.max_recv_data)
+        ot_send_estimate(
```

Comment:
yeah I think this would make sense, 2 questions:

1. Why is `EXTRA_OTS = 16384`? Is this the value for key exchange, prf and ghash?
2. Why is `ot_send_estimate` for `Prover` only `EXTRA_OTS`? Same for `ot_recv_estimate` and `Verifier`?